bin/deploy-storybook.sh

```diff
@@ -52,4 +55,10 @@ EOF
       s3://${STORYBOOK_BUCKET}/${STORYBOOK_PR_PATH}/index.html
 
   echo "Deployed Storybook PR redirect to ${STORYBOOK_PR_URL}"
+
+  curl \
+    -H'Content-Type: application/json' \
+    -H"Authorization: token $STORYBOOK_GITHUB_ACCESS_TOKEN" \
```

Comment:
FWIW, this access token is configured in the CircleCI settings and shouldn't be visible to public